developing/structure/transaction.md

```diff
@@ -0,0 +1,1152 @@
+# Transaction Structure
+
+Each block on a blockchain, including Cosmos-based ones like Osmosis, are constructed of a series
+of transactions.
+
+Each transaction, in turn, has its own internal structure. This document describes the structure
+of those transactions.
+
+## Retrieving a block
+
+You can retrieve a single block by any one of:
+
+* using the RPC API
+* using the REST API
+* running a full node
```

Comment:
running a full node and running osmosisd are synonymous right?

Reply:
Not quite. `osmosisd` is the binary. I can run it in mode to send a single transaction and exit `osmosisd tx ...`, or query something `osmosisd query ...` or in validator mode, or in full mode.